conn.go

```diff
@@ -72,3 +72,23 @@ func (conn *GrpcConnection) Start() error {
 func (conn *GrpcConnection) Handle() {
 	panic("implement me w/ test case :-)")
 }
+
+type Broadcaster interface {
+	Broadcast(data []byte, typ string, succCallback func(interface{}), errCallback func(error))
+}
+
+type ConnectionPool struct {
+	connMap map[ConnID]Connection
+}
+
+func (p *ConnectionPool) Broadcast(data []byte, typ string, succCallback func(interface{}), errCallback func(error)) {
```

Comment:
`succCallback`is so ambiguous. I recommend just `successCallback`

Reply:
I'm planning to remove callback functions